examples/otel-logs-truncate-splunk/README.md

```diff
@@ -0,0 +1,16 @@
+# Splunk HEC Example
```

Comment:
What's the difference between the existing `splunk-hec-<...>` and `otel-logs-<...>` examples and can this field not be added to an existing w/ documentation so that we don't need an additional one?

Reply:
not really thought through this. Maybe those examples also need to give live elsewhere. @theletterf wdyt?